R/post-body.R

```diff
@@ -24,3 +30,27 @@ parseBody <- function(body){
   }
   ret
 }
+
+#' @importFrom utils URLdecode
+#' @noRd
+parseMultipart <- function(body){
+  # This is not as generic as perhaps it should be, but should be relatively
+  # easy to extend if more binary formats are required
+  # Is there data in the request?
+  if (is.null(body) || length(body) == 0 || body == "") {
+    return(list())
+  }
+  # This is something odd with webutils::parse_multipart. For some reason it
+  # does not work when the boundary is set to the content type as per the
+  # webutils docs.
+  boundary <- "-----------------"
+  parsed_binary <- webutils::parse_multipart(body, boundary)
```

Comment:
There is nothing wrong with webutils. It was just poorly implemented on my part. The full boundary is part of the req$CONTENT_TYPE so we can fetch each current boundary like so:

    boundary <- strsplit(req$CONTENT_TYPE, split = "=")[[1]][2]